Prox/Prox/Map View/MapViewCardFooter.swift

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import SnapKit
+
+private let cardPadding = 10
+private let cardSpacing: CGFloat = 10
```

Comment:
Global constants/strings/etc. create a lot of bloat in files until it's hard to maintain, easy to loose track of unused values, and difficult to identify what is safe to re-use (and what just happens to be in a giant constants file). I prefer to keep the constants locally to the places they're used. A good example is our [FFAndroid styles](https://dxr.mozilla.org/mozilla-central/source/mobile/android/base/resources/values/styles.xml) which are so hard to maintain (most of these styles should really be embedded directly in the layout files). fwiw, I'm conflicted about our Colors.swift & Fonts.swift files.

Style.swift was intended to be for shared styles – should I rename it as such?